internal/server.go

```diff
@@ -5,27 +5,25 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-	"github.com/sarahhtbarton/microsoftleap/internal/helper"
+	"github.com/sarahhtbarton/microsoftleap/internal/matrix"
 )
 
 func Handler(w http.ResponseWriter, r *http.Request) {
 
 	vars := mux.Vars(r)
 
-	rows, columns, err1 := helper.ConvertMaptoInts(vars)
-	if err1 != nil {
-		http.Error(w, err1.Error(), http.StatusNotAcceptable)
+	rows, columns, err := helper.ConvertMaptoInts(vars)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotAcceptable)
 		return
 	}
-	
-	err2 := helper.ClientErrorHandling(rows, columns)
-	if err2 != nil {
-		http.Error(w, err2.Error(), http.StatusRequestedRangeNotSatisfiable)
+
+	matrix, err := matrix.GenerateFibMatrix(rows, columns)
```

Comment:
This is definitely going to cause a merge conflict. Your other PR moves this logic to a different package so I'd advise to merge that one into master first, then merge master into this branch and fix the inevitable conflicts